cmd/s2i/main.go

```diff
@@ -171,6 +171,7 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().StringVarP(&(cfg.Ref), "ref", "r", "", "Specify a ref to check-out")
 	buildCmd.Flags().StringVarP(&(cfg.AssembleUser), "assemble-user", "", "", "Specify the user to run assemble with")
 	buildCmd.Flags().StringVarP(&(cfg.ContextDir), "context-dir", "", "", "Specify the sub-directory inside the repository with the application sources")
+	buildCmd.Flags().StringVarP(&(cfg.ExcludeRegExp), "exclude", "", "", "Regular expression for selecting files from the source tree to exclude from the tar stream; see https://golang.org/pkg/regexp for syntax")
```

Comment:
no, but the idea here was to kill two birds w/ one stone:
1) give users a way to include .git
2) give users a way to exclude other arbitrary files

given that this seems to be a relatively rare use case, i'm ok with the solution to (1) being slightly more complicated, in exchange for solving both problems, rather than just solving the single problem.  But i do understand your concern and share it...  maybe we need a tie breaker.  @gabemontero ?